app/views/user_mailer/monthly_owner_wrapup.html.slim

```diff
@@ -0,0 +1,44 @@
+p Hello #{@owner.display_name},
+
+p Another month has passed.  Give your collaborators some love and encourage them to come back by sending a monthly status email.  The email could include:
+ul
+  li What they've accomplished this month
+  li What you'll be using the transcribed material for
+  li call to action - "Know anyone else who would be interested in working on this collection?  Invite them to join in on the fun."
+  li Here's an example of a monthly email: 
+  span =link_to "http://content.fromthepage.com/crowdsourcing-project-newsletter", 'http://content.fromthepage.com/crowdsourcing-project-newsletter', target: '_blank', rel: 'nofollow'
+p Here are the stats for all of your collections in the past month:
+
+table
+  tr
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Work'.pluralize(@owner.work_count)}: #{number_with_delimiter @owner.work_count} 
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Page'.pluralize(@owner.page_count)}: #{number_with_delimiter @owner.page_count} 
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Note'.pluralize(@owner.comment_count(@start_date, @end_date))}: #{number_with_delimiter @owner.comment_count(@start_date, @end_date)} 
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Collaborator'.pluralize(@owner.contributor_count(@start_date, @end_date))}: #{number_with_delimiter @owner.contributor_count(@start_date, @end_date)} 
+  tr
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Page'.pluralize(@owner.transcription_count(@start_date, @end_date))} transcribed: #{number_with_delimiter @owner.transcription_count(@start_date, @end_date)}
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Page'.pluralize(@owner.edit_count(@start_date, @end_date))} edited: #{number_with_delimiter @owner.edit_count(@start_date, @end_date)}
+    td(style="text-align: center; width: 25%; padding: 0 10px 0;")
+      h3 #{'Page'.pluralize(@owner.translation_count(@start_date, @end_date))} translated: #{number_with_delimiter @owner.translation_count(@start_date, @end_date)} 
```

Comment:
Can we wrap this in logic and only show it if it's turned on?  (Can we do conditionals within tables?)

Reply:
What do you mean if it's turned on?  (We can do conditionals within tables, but I don't understand the question.)

Reply:
Translation isn't enabled for most collections.

Reply:
In fact, none on ftp.com that I know of.

Reply:
Got it.

Reply:
It makes the table look kind of weird, though:

![email-table](https://user-images.githubusercontent.com/16854029/38102849-848476ca-3352-11e8-8f37-545d26bc4afc.PNG)

I have no idea if there's a way to make those two cells center themselves if the other cells are hidden.